frontend/src/app/testResults/TestResultsList.jsx

```diff
@@ -35,12 +49,10 @@ const TestResultsList = () => {
     return <p>Loading</p>;
   }
   if (error) {
-    console.error(error);
-    return (
-      <div>
-        <h3>There was an error:</h3>
-      </div>
-    );
+    appInsights.trackEvent({
+      name: "Failed Fetching Tests Results",
+    });
+    return error;
```

Comment:
Same question here, can we return an `Error` object or do we need to `throw` it?